DeAceroDLabs/marketplace-frontend
# Feature/dmp 478

## What is done

[[DMP-478]  Loading signup](https://deacero.atlassian.net/browse/DMP-478)

## How is done

- Added a `sending` button with `Enviando...`  text, this button will be shown only if there are no errors, if errors, `Enviar solicitud` button will continue to be displayed, this last button checks if there are no errors.

## How to test

1. Run the project via `npm start`
2. signup
3. Fill everything with errors in the last form, the `Enviar solicitud` button should be there.
4. Fill everything correctly in the last form, the `Enviando...` button should be there.
